src/models/regression.py

```diff
@@ -235,6 +235,9 @@ def _calculate_big_mean(self) -> np.ndarray:
                 # for us its + 2, since 0 is not a class either
                 pred_months_onehot = np.eye(14)[pred_months][:, 1:-1]
                 x_in = np.concatenate((x_in, pred_months_onehot), axis=-1)
+            if self.experiment == 'nowcast':
```

Comment:
What does the `_calculate_big_mean()` function actually do? Calculate the mean across the whole dataset (incl. the current timestep??) to act as a baseline for the calculation of SHAP values?

Reply:
It calculates the variable-wise mean of the training data. Yes, this is to act as a baseline for the shap values (in the `explain` function, you can see that the output of this function is passed to the linear explainer).